packages/firestore/src/platform/rn/base64.ts

```diff
@@ -16,6 +16,7 @@
  */
 
 import { base64 } from '@firebase/util';
+import { uint8ArrayFromBinaryString } from '../../util/byte_string';
```

Comment:
I see Github Actions has inlined this ts error from the build log ('uint8ArrayFromBinaryString' is defined but never used) so that's cool.

Reply:
Removed this line.